src/OpenLoco/src/Vehicles/Orders.cpp

```diff
@@ -108,7 +96,7 @@ namespace OpenLoco::Vehicles
 
     uint32_t Order::getOffset() const
     {
-        return this - _orderTable;
+        return this - reinterpret_cast<Order*>(OrderManager::orders());
```

Comment:
orders() should really return a Order* not a uint8_t*

Reply:
You're right, that would be better. I was going by the type in the GameState, which I supposed made sense because not all orders are of the same length. I'll not amend the GameState, for now, though, and just move the static_cast in `orders()`.